src/app/components/utility/modal/modal.component.html

```diff
@@ -1,42 +0,0 @@
-<div class="modal-container">
```

Comment:
@Suryansh5545 Why have you deleted this file?

Reply:
where isn't a need to change anything in it

Reply:
It's scss file and you have already reverted that file.

Reply:
should i repush it

Reply:
i am re pushing it

Reply:
i push the model.component file again

Reply:
without any change from the original one